java/integTest/src/test/java/glide/cluster/ClusterTransactionTests.java

```diff
@@ -92,4 +96,20 @@ public void lastsave() {
         var response = clusterClient.exec(new ClusterTransaction().lastsave()).get();
         assertTrue(Instant.ofEpochSecond((long) response[0]).isAfter(yesterday));
     }
+
+    @Test
+    @SneakyThrows
+    public void zrank_zrevrank_withscores() {
+        assumeTrue(REDIS_VERSION.isGreaterThanOrEqualTo("7.2.0"));
+        String zSetKey1 = "{key}:zsetKey1-" + UUID.randomUUID();
```

Comment:
Is the hash slot needed here? If not should we remove it from the key name?

Reply:
it's okay to keep it.